benefits/templates/error-base.html

```diff
@@ -0,0 +1,26 @@
+{% extends "core/base.html" %}
+{% load i18n %}
```

Comment:
Maybe this is unnecessary???

Reply:
Seems like it, there are no `{% translate %}` tags on this base page.

Reply:
Done